### DISCUSSION TITLE
Is it possible to add mass update in GUI mode

### DISCUSSION THREAD

**Discussion by Kaien715** [comictagger/comictagger#452] [General]
Hi there

Firstable, thank you so much for such wonderful utilities. For years I have been trying to find a way to organize my manga. Comictagger truely help me so much.
Because my manga are all in Chinese, the auto identify option isn't much of use for me. I have to manually enter the tag myself. Most of my time is adding same Author or publisher name to multiple books. It would be really grateful if Comictagger can mass updating all selected books with same attribute (like a tag in credit).

I have read this is possible to be done in command line approach, but I have yet to figure it out...

**Comment by lordwelch** [comment]
You are not alone it is a common feature request #23.
I don't currently have any plans for this, I'm working on other parts of ComicTagger. Unless someone else contributes a pull request you will more than likely be able to auto-identify your Chinese manga before a batch update GUI is available.
